python/poetry/Portfile

```diff
@@ -45,29 +43,37 @@ foreach pv {310 39 38 37} {
     }
 }
 
+python.pep517           yes
+python.pep517_backend
```

Comment:
I guess it doesn't matter that much since there's a dependency on poetry-core added below, but should this be set to `poetry`?